pkg/metaserver/util/util.go

```diff
@@ -70,20 +104,33 @@ func UnsafeResourceToKind(r string) string {
 }
 
 func UnsafeKindToResource(k string) string {
```

Comment:
Can the method here be handled directly with `meta.UnsafeGuessKindToResource`? 
I see that the call place contains the complete `gvk`.

Reply:
@zc2638  Not really good, because meta.UnsafeGuessKindToResource() only contains a few examples of specially transformed resources. According to its logic, resources such as Gateway(resource name is gateways) will be transformed incorrectly. We need to deal with the specially transformed resources based on it.